src/main/java/org/myrobotlab/service/GoogleSearch.java

```diff
@@ -385,12 +384,12 @@ public static void main(String[] args) {
 
   @Override
   public String getLanguage() {
-    return locale.getLanguage();
+    return Runtime.getInstance().getLanguage();

```

Comment:
This means you can't search in any language other than the current runtime language.

Reply:
good catch .. these needed to be removed